js/main.js

```diff
@@ -630,16 +656,270 @@ function showCluster(a) {
+            $("#variables").append(new Option(attr, attr));
+    }
+}
+
+
+function addExamplesForSearch(e){
+    var firstElem;
+    for(elem in sigInst._core.graph.nodesIndex){
+        firstElem = elem;
+        break;
+    }
+    if(isNaN(firstElem))
+        firstElem = String(firstElem);
+    var b = sigInst._core.graph.nodesIndex[firstElem];
+    
+    var f = b.attr;
+
+    $("#containerForSlider").hide();
+    $("#myVariables").hide();
+
+    if(e.target.value == "Phenotype Name"){
+        $("#myVariables").val("e.g. diabetes");
+        $("#myVariables").show();
+    }
+
+    else{
+        if (f.attributes) {
+            for (var attr in f.attributes){
+                if(attr == e.target.value){
+                    var exampleValue = f.attributes[attr];
+                  
+                    // If the attribute we're searching for is a number, we need to find the min and max so that
+                    //   we can set the range of our slider input
+                    if(!isNaN(exampleValue)){
+                        //we'll have to go through iterNodes to find all possible values for this attribute
+                        smallestValue = 100000
+                        largestValue = -1
+                        sigInst.iterNodes(function (a) {
+                            var d = Number(a.attr.attributes[e.target.value]);
+                            if(d < smallestValue)
+                                smallestValue = d
+                            if(d > largestValue)
+                                largestValue = d
+                        });
+
+                        smallestValue = Math.floor(smallestValue* 100)/100;
+                        largestValue = Math.ceil(largestValue* 100)/100;
+
+                        slider.destroy();
+                        $("#containerForSlider").show();
+
+                        if(Number.isInteger(largestValue) && largestValue != 1 && largestValue <= 100){
+                            slider = new rSlider({target: '#slider', 
+                                                values: {min: smallestValue, max: largestValue},
+                                                step: 1,
+                                                range: true, 
+                                                scale: false, 
+                                                labels: false});
+
+                        } else {
+                            slider = new rSlider({target: '#slider', 
+                                                values: {min: smallestValue, max: largestValue},
+                                                step: (largestValue - smallestValue)/100,
+                                                range: true, 
+                                                scale: false, 
+                                                labels: false});
+                        }
+                        
+                        $("#myVariables").style("visibility", "hidden");
```

Comment:
there is no ".style" in js. it should be ".css".  But we can remove the line its not doing anything

Reply:
Alright, I am removing it.